bson/src/test/unit/org/bson/codecs/DocumentCodecSpecification.groovy

```diff
@@ -57,8 +58,12 @@ import static org.bson.UuidRepresentation.STANDARD
 import static org.bson.UuidRepresentation.UNSPECIFIED
 import static org.bson.codecs.configuration.CodecRegistries.fromCodecs
 import static org.bson.codecs.configuration.CodecRegistries.fromProviders
+import static org.bson.codecs.configuration.CodecRegistries.fromRegistries
 
 class DocumentCodecSpecification extends Specification {
+    static final CodecRegistry REGISTRY = fromRegistries(fromCodecs(new UuidCodec(JAVA_LEGACY)),
```

Comment:
Why set REGISTRY only to later use withUuidRepresentation and the same JAVA_LEGACY representation?

Reply:
Yeah, that is confusing.  It's because calling withUuidRepresentation on the DocumentCodec won't change anything about the other codecs in the registry, in particular the UuidCodec.

But I will change everything to STANDARD from JAVA_LEGACY since if we're going to pick one it might as well be STANDARD.

Reply:
Done